data-prepper-plugins/s3-sink/src/integrationTest/java/org/opensearch/dataprepper/plugins/sink/s3/S3SinkServiceIT.java

```diff
@@ -152,6 +155,7 @@ public void setUp() {
         s3region = System.getProperty("tests.s3sink.region");
 
         s3Client = S3Client.builder().region(Region.of(s3region)).build();
```

Comment:
can we remove s3Client ?

Reply:
S3Client is used in the test to validate objects